### ISSUE TITLE
Clean-up releases et al.

### ISSUE THREAD

**Issue by marcelstoer** [gitlab4j/gitlab4j-api#884]
I would appreciate if you could address a couple of issues around releases:

- Create new release to make the fixes recently merged available to the general public.
- 5.0 and 5.0.1 are available on Maven Central but they don't have any representation here on GitHub. The tags are here but not the corresponding releases.
- Ever since 84129c51e0b5bc7cb9185deef2ff1af8a62c9a1c the CI builds are failing (see screen shot below).

<img width="589" alt="Screen Shot 2022-12-13 at 09 03 25" src="https://user-images.githubusercontent.com/624195/207259981-b23a4725-74c6-4d45-8910-d646b27e3c3b.png">

**Comment by jabby** [comment]
I'm working on this. Thanks for your reminder. 

- [x] create 5.0.0 and 5.0.1 releases
- [ ] investigate for failing builds
- [ ] create new release

**Comment by jmini** [comment]
@marcelstoer I only joined the project very recently, but for me we the points that you have raised in this issue are OK for now.

* The CI on GitHub looks stable to me, travis was removed.
    * For the CI we might use testcontainer to have a newer version of GitLab started during the tests: #925
* New releases were created and the corresponding release pages exist.
    * For the release we might investigate JReleaser which could help with the automation of even more tasks: #923 
* The main pain point of not being compatible with Spring Boot 3 is being worked on with the [branch `6.x`](https://github.com/gitlab4j/gitlab4j-api/tree/6.x): #926
    * A first RC for was created to get feedback from Spring users: `6.0.0-rc.1`
* I have tried to give feedback on each of the pending PR

I agree not everything is perfect (this project stays a side project when it could get somebody working full time on it for a while), but for me this issue can be closed.
